tools/images_packer/__main__.py

```diff
@@ -69,6 +69,13 @@ def main_build_offline_ota_image_bundle(args: argparse.Namespace):
                     image_version=_image_version,
                 )
             )
+            if _ecu_id in image_files:
```

Comment:
I think this should not be warning. For the user, this is the unexpected behavior, so it should be error.

Reply:
Yes I agree! This is unexpected behavior. Let me fix it.

Reply:
Updated at [786ef4d](https://github.com/tier4/ota-client/pull/267/commits/786ef4d160af649bf21ca82f42b5e30b34eca644), now multiple images specified to the same ECU will cause the images_packer logging an error and then exit.